Testing System/src/main/webapp/WEB-INF/web.xml

```diff
@@ -13,4 +13,54 @@
         <servlet-name>add</servlet-name>
         <url-pattern>/add</url-pattern>
     </servlet-mapping>
+
+    <servlet>
```

Comment:
Possible simplification: All these descriptions could be replaced by `@WebServlet("/YOUR_PATH")` annotation above the Servlet class (`import javax.servlet.annotation.WebServlet`.